### PR TITLE
BLE test Apps

### PR DESCRIPTION
**Pull request by Skorpionm**

### CODE REVIEW

File: applications/system/ble_ibeacon/helpers/ble_ibeacon_app.c [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+#endif
+             ),
+        .config_feature_bit_map =
+            (SL_SI91X_FEAT_SLEEP_GPIO_SEL_BITMAP | SL_SI91X_ENABLE_ENHANCED_MAX_PSP)}};
+
+typedef enum {
+    HELP,
+    HELP_HELP,
+
+    ble_ibeacon_COMMANDS_MAX,
```
Comment (gsurkov) [inline applications/system/ble_ibeacon/helpers/ble_ibeacon_app.c]: Use CamelCase for enum members

File: applications/system/ble_test/helpers/ble_test_app.c [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+#if RSI_BLE_DISABLE_CODED_PHY_FROM_HOST
+             | (SL_SI91X_BLE_DISABLE_CODED_PHY_FROM_HOST) //Disable Coded PHY from app
+#endif
+#if BLE_SIMPLE_GATT
+             | SL_SI91X_BLE_GATT_INIT
+#endif
+             ),
+        .config_feature_bit_map = (SL_SI91X_FEAT_SLEEP_GPIO_SEL_BITMAP)}};
+
+typedef enum {
```
Comment (gsurkov) [inline applications/system/ble_test/helpers/ble_test_app.c:97]: Use CamelCase for enum members

**Review by gsurkov — CHANGES_REQUESTED** [review]

File: applications/system/ble_test/helpers/ble_test_app.c (hunk 2 above)
Comment (Skorpionm) [reply applications/system/ble_test/helpers/ble_test_app.c:97]: fix

File: applications/system/ble_ibeacon/helpers/ble_ibeacon_app.c (hunk 1 above)
Comment (Skorpionm) [reply applications/system/ble_ibeacon/helpers/ble_ibeacon_app.c]: fix

**Review by gsurkov — APPROVED** [review]

**Pull request merged**